src/javascript/app_2/App/Components/Form/TimePicker/time_picker.jsx

```diff
@@ -0,0 +1,123 @@
+import { observer }       from 'mobx-react';
+import PropTypes          from 'prop-types';
+import React              from 'react';
+import TimePickerDropdown from './time_picker_dropdown.jsx';
+import InputField         from '../input_field.jsx';
+
+class TimePicker extends React.PureComponent {
+    constructor (props) {
+        super(props);
+        this.state = {
+            is_open: false,
+            value  : '',
+        };
+    }
+
+    componentDidMount() {
+        document.addEventListener('mousedown', this.handleClickOutside);
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener('mousedown', this.handleClickOutside);
+    }
+
+    toggleDropDown = () => {
```

Comment:
I suggest we rename this to `toggleTimePicker` as we already have a `DropDown` component.